docs/data/routes/docs/nextjs/tracking-and-analytics/configuration/en.md

```diff
@@ -0,0 +1,66 @@
+---
+name: configuration
+routeTemplate: ./data/component-templates/article.yml
+title: Enabling tracking and analytics
+---
+# Walkthrough: Enabling tracking and analytics
+
+This topic will guide you through the steps required to enable full Sitecore tracking and analytics for your Next.js application.
+
+> Note this does not cover client-side tracking via the JSS tracking API, which is possible for *both SSG and SSR*. Please see the [JSS Tracking](/docs/fundamentals/services/tracking) page for details.
+
+## Next.js application
+
+### SSR only
+
+You must use a Next.js SSR page route. An example SSR route is included with the Next.js sample app. See [this page](/docs/nextjs/page-routing/switching-to-ssr) for details.
+
+### Layout Service requests
+
+Sitecore JSS layout service requests must:
+
+1. have tracking enabled (default), and
+2. perform [header passing](/docs/nextjs/tracking-and-analytics/overview#header-passing) (default with the Next.js sample application)
+
+Both of these are taken care of by using the `RestLayoutService` included with the Next.js SDK (part of the `@sitecore-jss/sitecore-jss-nextjs` npm package). 
```

Comment:
Maybe link to the RestLayoutServie code in GitHub?

Reply:
Linking to sample app source code makes sense, but I'm leery to link to package source code. Looks like we've avoided it in the docs thus far, think I might skip that here.